src/vm/olevariant.cpp

```diff
@@ -755,7 +755,7 @@ UINT OleVariant::GetElementSizeForVarType(VARTYPE vt, MethodTable *pInterfaceMT)
         8,                      // VT_UI8
         4,                      // VT_INT 
         4,                      // VT_UINT
-        0,                      // VT_VOID
+        1,                      // VT_VOID
```

Comment:
Why was this needed?

Reply:
I had an assert somewhere while I was writing the GetNativeVariantForVarType function that verified that the native size of the returned type was equal to the native size returned from GetElementSizeForVarType and I was using System.Void for VT_VOID, which is technically size 1. I can't seem to find where I put that assert, so I can probably revert this.

Reply:
I am fine leaving this if need be, but `sizeof(void)` in C is undefined so if we are relying on .NET's interpretation of that size please comment as such.

Reply:
I decided to revert it for now since I didn't see any need to change it.